app/models/service_template_transformation_plan_task.rb

```diff
@@ -86,17 +94,17 @@ def transformation_type
   def virtv2v_disks
     return options[:virtv2v_disks] if options[:virtv2v_disks].present?
 
-    options[:virtv2v_disks] = calculate_virtv2v_disks
-    save!
+    updates = {:virtv2v_disks => calculate_virtv2v_disks}
+    update_options(updates)
```

Comment:
Probably don't need the local var here unless you're using it later and I'm missing it, can be `update_options(:virtv2v_disks => ....)`

Reply:
yep, missed that